sdv/data_processing/data_processor.py

```diff
@@ -725,6 +833,10 @@ def reverse_transform(self, data, reset_keys=False):
         except rdt.errors.NotFittedError:
             LOGGER.info(f'HyperTransformer has not been fitted for table {self.table_name}')
 
+        for transformer in self.columns_to_multi_col_transformer.values():
```

Comment:
I needed this to make the RandomLocation work with SDV. I'm down to discuss other options or changes we could do on the `reverse_transform` of the DataProcessor. Basically, without this change, it was not crashing but the `address columns` were not generated because the `hyper transformer` were never called for these columns, mainly because this transformer doesn't have any `output_columns`.

Reply:
should we mark the `RandomLocationGenerator` as a generator? Then it should generate the columns when `create_anonymized_columns` is called below

Reply:
That's a good idea but  `create_anonymized_columns` will have to be updated to work with multi-column transformers. Especially we will have to change this [line](https://github.com/sdv-dev/RDT/blob/c2c70646563045f998981e15c4cbe60662cc42ed/rdt/hyper_transformer.py#L716).

Reply:
Maybe we should?